Documentation cleanup and rename Ternary to ConditionalOperator (#1066)
hyperledger/solang: docs/targets/solana.rst

- An address literal has to be specified using the ``address"36VtvSbE6jVGGQytYWSaDPG7uZphaxEjpJHUUpuUbq4D"`` syntax
- There is no ``ecrecover()`` builtin function, but there is a ``signatureVerify()`` function which can check ed25519
  signatures.
- There is no concept of gas, so there is no gas functions
- Solana balance is stored in a ``uint64``, so *address* ``.balance``, ``.transfer()`` and ``.send()``
  all use ``uint64`` rather than ``uint256``.
- Solana uses different accounts for the program code, and the contract data.
- Programs are upgradable. There is no need to implement upgrades in Solidity.

> - Different builtins are different.

Reviewer: Please, remove this line.

Author: I've changes it to:
- Solana provides different builtins, e.g. ``tx.program_id`` and ``tx.accounts``.

.. note::

>     A naive way to implement this might be to let the caller transfer

Reviewer 2: ```suggestion
    A naive way to implement this is to let the caller transfer
```

______________

In Solang's Github repository, there is a directory called ``solana-library``. It contains libraries for Solidity contracts

> to interact with Solana specific instructions. There are two libraries there: one for SPL tokens and another

Reviewer 2: ```suggestion
to interact with Solana specific instructions. We provide two libraries: one for SPL tokens and another
```
